docker/Dockerfile.dmqnode

```diff
@@ -1,26 +1,62 @@
 # Duplicate definition of DIST before FROM is needed to be able to use it
 # in docker image name:
 ARG  DIST=xenial
-FROM sociomantictsunami/develdlang:$DIST-v6 as builder
+FROM sociomantictsunami/develdlang:$DIST-v7 as builder
 # Copies the whole project as makd needs git history:
 COPY . /project/
 WORKDIR /project/
 # Redefine arguments as env vars to be used inside build.sh script:
 ARG DIST=xenial
 ARG DMD=2.078.*
-ENV DMD=$DMD DIST=$DIST
-RUN docker/build.sh
+ENV DMD=$DMD
+RUN set -eu \
+    # Compiler configuration \
+    && . submodules/beaver/lib/dlang.sh \
+    \
+    && set_dc_dver \
+    \
+    # Install dependencies \
+    \
+    && case "$DMD" in \
+        dmd*   ) PKG= ;; \
+        1.*    ) PKG="dmd1=$DMD-$DIST" ;; \
+        2.*.s* ) PKG="dmd-transitional=$DMD-$DIST" ;; \
+        2.*    ) PKG="dmd-compiler=$DMD dmd-tools=$DMD libphobos2-dev=$DMD" ;; \
+        *      ) echo "Unknown \$DMD ($DMD)" >&2; exit 1 ;; \
+    esac \
+    \
+    && apt update \
+    && apt install -y --allow-downgrades \
+        $PKG \
+        libebtree6-dev \
+        liblzo2-dev \
+        libglib2.0-dev \
+        libpcre3-dev \
+        libgcrypt-dev \
+        libgpg-error-dev \
+    \
+    # Build app \
+    && if_d 2 make d2conv \
+    && make all pkg F=production
 
 ARG DIST=xenial
 # For now plain ubuntu image is used as base for simplicity. It certainly can be
 # optimized to bare minimum but right now it is not important:
-FROM sociomantictsunami/runtimebase:$DIST-v6
+FROM sociomantictsunami/runtimebase:$DIST-v7
+
+# Set up directories and install the node
 COPY --from=builder /project/build/production/pkg/ /packages/
-COPY docker/install.sh /tmp/
-RUN /tmp/install.sh && rm /tmp/install.sh
-# Update node folder with relevant config to finalize installation
-COPY ./doc/etc/config.docker.ini /srv/dmqnode/dmqnode-0/etc/config.ini
+RUN set -eu \
+    && apt update && apt install -y libebtree6 \
+    && mkdir /etc/dmqnode \
+    && mkdir -p /srv/dmqnode/dmqnode-0/log \
+    && ln -s /etc/dmqnode /srv/dmqnode/dmqnode-0/etc \
+    && apt install -y /packages/*.deb
```

Comment:
Same here. The only interesting part is you are installing the built package (and the run-time dependencies you want to install).